package.json

```diff
@@ -4,9 +4,13 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "start:prod": "node --require @babel/register src/index.js",
```

Comment:
I think this script is not necessary you can use the same production script.

Reply:
I somehow mussed that script. Apologies.